nipype test failed at test_utils.py with free surfer 6.0.0

### Summary
I tried to set up all nipype required dependencies, and run.

``` python
import nipype
nipype.config.set('logging', 'workflow_level',  'CRITICAL')
nipype.config.set('logging', 'interface_level', 'CRITICAL')

nipype.logging.update_logging(nipype.config)

# Run the test: Increase verbosity parameter for more info
nipype.test()
```

### Actual behavior

``` python 
.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/freesurfer/tests/test_utils.py ....
F

========================================================= FAILURES ==========================================================
tmpdir = local('/tmp/pytest-of-qinqian/pytest-10/test_mrisexpand0')                                                  [28/664]

    @pytest.mark.skipif(fs.no_freesurfer(), reason="freesurfer is not installed")
    def test_mrisexpand(tmpdir):
        fssrc = FreeSurferSource(subjects_dir=fs.Info.subjectsdir(),
                                 subject_id='fsaverage', hemi='lh')

        fsavginfo = fssrc.run().outputs.get()

        # dt=60 to ensure very short runtime
        expand_if = fs.MRIsExpand(in_file=fsavginfo['smoothwm'],
                                  out_name='expandtmp',
                                  distance=1,
                                  dt=60)

        expand_nd = pe.Node(
            fs.MRIsExpand(in_file=fsavginfo['smoothwm'],
                          out_name='expandtmp',
                          distance=1,
                          dt=60),
            name='expand_node')

        # Interfaces should have same command line at instantiation
        orig_cmdline = 'mris_expand -T 60 {} 1 expandtmp'.format(fsavginfo['smoothwm'])
        assert expand_if.cmdline == orig_cmdline
        assert expand_nd.interface.cmdline == orig_cmdline

        # Run both interfaces
        if_res = expand_if.run()
        nd_res = expand_nd.run()

        # Commandlines differ
        node_cmdline = 'mris_expand -T 60 -pial {cwd}/lh.pial {cwd}/lh.smoothwm ' \
            '1 expandtmp'.format(cwd=nd_res.runtime.cwd)
        assert if_res.runtime.cmdline == orig_cmdline
        assert nd_res.runtime.cmdline == node_cmdline

        # Check output
        if_out_file = if_res.outputs.get()['out_file']
        nd_out_file = nd_res.outputs.get()['out_file']
        # Same filename
        assert op.basename(if_out_file) == op.basename(nd_out_file)
        # Interface places output in source directory
        assert op.dirname(if_out_file) == op.dirname(fsavginfo['smoothwm'])
        # Node places output in working directory
        assert op.dirname(nd_out_file) == nd_res.runtime.cwd

        # Remove test surface
>       os.unlink(if_out_file)
E       OSError: [Errno 2] No such file or directory: '/usr/local/freesurfer/subjects/fsaverage/surf/lh.expandtmp'

./.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/freesurfer/tests/test_utils.py:212: OSError
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
===================================== 1 failed, 1079 passed, 5 skipped in 68.30 seconds ===================

``` 


### How to replicate the behaviour
use the latest nipype-1.0.0.dev0+gf85cf0d from github, and install with `python setup.py install --user` with python2.7 and free surfer  v6.0.0.

When running with root, the test get more "passed" case, but with several failure and warnings, as follows:

``` python
===================================================== warnings summary ======================================================
workflows/dmri/fsl/artifacts.py::nipype.workflows.dmri.fsl.artifacts.all_fmb_pipeline
  /home/qinqian/.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/fsl/preprocess.py:1
597: UserWarning: This has not been fully tested. Please report any failures.
    warn('This has not been fully tested. Please report any failures.')
  /usr/local/lib/python2.7/dist-packages/_pytest/warnings.py:82: UnicodeWarning: Warning is using unicode non convertible to ascii, converting to a safe representation:
    /home/qinqian/.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/fsl/preprocess.py
:1597: UserWarning: This has not been fully tested. Please report any failures.
    warn('This has not been fully tested. Please report any failures.')

    UnicodeWarning)

workflows/dmri/fsl/artifacts.py::nipype.workflows.dmri.fsl.artifacts.sdc_fmb
  /home/qinqian/.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/fsl/preprocess.py:1
597: UserWarning: This has not been fully tested. Please report any failures.
    warn('This has not been fully tested. Please report any failures.')

workflows/dmri/fsl/epi.py::nipype.workflows.dmri.fsl.epi.create_dmri_preprocessing
  /home/qinqian/.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/fsl/preprocess.py:1
597: UserWarning: This has not been fully tested. Please report any failures.
    warn('This has not been fully tested. Please report any failures.')

workflows/dmri/fsl/epi.py::nipype.workflows.dmri.fsl.epi.create_dmri_preprocessing
  /home/qinqian/.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/fsl/preprocess.py:1
597: UserWarning: This has not been fully tested. Please report any failures.
    warn('This has not been fully tested. Please report any failures.')

workflows/dmri/fsl/epi.py::nipype.workflows.dmri.fsl.epi.create_epidewarp_pipeline
  /home/qinqian/.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype/interfaces/fsl/preprocess.py:1
597: UserWarning: This has not been fully tested. Please report any failures.
    warn('This has not been fully tested. Please report any failures.')

-- Docs: http://doc.pytest.org/en/latest/warnings.html
============================= 2674 passed, 45 skipped, 7 xfailed, 5 warnings in 297.61 seconds ==============================

```

It seems that some parts of nipype environment is not installed or configured successfully, can you provide me some suggestions?  Thanks.

### Platform details:
{'nibabel_version': '2.1.0', 'sys_executable': '/usr/bin/python', 'network_version': '1.11', 'numpy_version': '1.13.1', 'sys_platform': 'linux2', 'sys_version': '2.7.12 (default, Nov 19 2016, 06:48:10) \n[GCC 5.4.0 20160609]', 'commit_source': u'installation', 'commit_hash': u'f85cf0d', 'pkg_path': './.local/lib/python2.7/site-packages/nipype-1.0.0.dev0+gf85cf0d-py2.7.egg/nipype', 'nipype_version': u'1.0.0-dev', 'traits_version': '4.6.0', 'scipy_version': '0.19.1'}
1.0.0-dev
### Execution environment
Ubuntu 16.04.

--- a/nipype/interfaces/freesurfer/tests/test_utils.py
+++ b/nipype/interfaces/freesurfer/tests/test_utils.py
@@ -188,18 +188,16 @@ def test_mrisexpand(tmpdir):
     assert expand_if.cmdline == orig_cmdline
     assert expand_nd.interface.cmdline == orig_cmdline
 
-    # Run both interfaces
-    if_res = expand_if.run()
+    # Run Node interface
     nd_res = expand_nd.run()
 
     # Commandlines differ
     node_cmdline = 'mris_expand -T 60 -pial {cwd}/lh.pial {cwd}/lh.smoothwm ' \
         '1 expandtmp'.format(cwd=nd_res.runtime.cwd)
-    assert if_res.runtime.cmdline == orig_cmdline
     assert nd_res.runtime.cmdline == node_cmdline
 
     # Check output
-    if_out_file = if_res.outputs.get()['out_file']
+    if_out_file = expand_if._list_outputs()['out_file']
     nd_out_file = nd_res.outputs.get()['out_file']
     # Same filename
     assert op.basename(if_out_file) == op.basename(nd_out_file)
@@ -207,6 +205,3 @@ def test_mrisexpand(tmpdir):
     assert op.dirname(if_out_file) == op.dirname(fsavginfo['smoothwm'])
     # Node places output in working directory
     assert op.dirname(nd_out_file) == nd_res.runtime.cwd
-
-    # Remove test surface
-    os.unlink(if_out_file)

